.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    directory: "/"
+    open-pull-requests-limit: 20
+    groups:
+      minor-patch-dependencies:
+        patterns:
+          - "*"
+        update-types:
+          - "minor"
+          - "patch"
+    schedule:
+      interval: "monthly"
+      time: "06:00"
+      timezone: "America/New_York"
+    target-branch: "main"
+    reviewers:
+      - "@DataBiosphere/platform-foundation-codeowners"
+    labels:
+      - "dependency"
+      - "gradle"
+    commit-message:
+      prefix: "[PF-2983]"
```

Comment:
Long-lived ticket, https://broadworkbench.atlassian.net/browse/PF-2983

Reply:
Something we like to do is add labels which make the PR list a little more visual when viewing https://github.com/pulls/review-requested

![Screenshot 2024-02-09 at 11 57 08 AM](https://github.com/DataBiosphere/terra-common-lib/assets/116679/c7d42d1c-da34-47f9-b95d-59137a71e54b)

There are other times when we manually do library updates so we reserve that prefix for dev-initiated PRs and a slightly different one for dependabot-initiated PRs.

Reply:
@rushtong does the automatic linking to the JIRA ticket work with "-dependabot" appended?

Reply:
It does link in our board.

Reply:
OK, we can always tweak the prefix if people think it helps (I've always found that the titles and labels make it pretty clear).